gio-subclass/src/auto/action.rs

```diff
@@ -120,17 +120,17 @@ unsafe extern "C" fn action_get_name<T: ObjectType>
     
     match rs_ret {
         Some(t_ret) => {
-            let ret = t_ret.to_glib_full();
-            unsafe extern "C" fn destroy_ret(p: glib_ffi::gpointer){
-                String::from_glib_full(p as *const c_char);
-            };
-            gobject_ffi::g_object_set_qdata_full(gptr as *mut gobject_ffi::GObject,
-                glib_ffi::g_quark_from_string("rs_action_get_name_ret".to_glib_none().0),
-                ret as *mut c_void,
-                Some(destroy_ret)
-            );
-            ret
-        },
+        let ret = t_ret.to_glib_full();
+        unsafe extern "C" fn destroy_ret(p: glib_ffi::gpointer){
+            let _:String = String::from_glib_full(p as *const c_char);
```

Comment:
Is the type annotation needed here even if you explictly call the specific impl of the trait?

Should get some more spaces though `let _: String = ...`

Reply:
The type annotation is not needed for this one, but for ```Vec<String>``` it was. I just left it there for all.